fb_apt/attributes/default.rb

```diff
@@ -0,0 +1,22 @@
+# vim: syntax=ruby:expandtab:shiftwidth=2:softtabstop=2:tabstop=2
+
+if node.debian?
+  mirror = 'http://http.debian.net/debian'
+elsif node.ubuntu?
+  mirror = 'http://archive.ubuntu.com/ubuntu'
+end
```

Comment:
Looked into this, it's actually non-trivial as mint is layered on top of ubuntu (they don't provide full mirrors for it), so we'd need some logic to map the mint -> ubuntu codenames. I'll leave this out for now, but feel free to add it in a separate PR if you want.